src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/DeleteManagedOrganizationUserAccountCommand.cs

```diff
@@ -114,7 +117,9 @@ public async Task DeleteUserAsync(Guid organizationId, Guid organizationUserId,
             }
         }
 
-        await DeleteManyAsync(users);
+        var usersToDelete = users.Where(user => results.Select(r => r.OrganizationUserId == user.Id && string.IsNullOrEmpty(r.ErrorMessage)).Count() > 1);
```

Comment:
Non-blocking, fyi only: the idiomatic way to write this is:
```suggestion
        var usersToDelete = users.Where(user => results.Any(r => r.OrganizationUserId == user.Id && string.IsNullOrEmpty(r.ErrorMessage)));
```

It's the equivalent of `Array.some` in JS/TS.